lib/plugins/api.template.js

```diff
@@ -61,14 +45,12 @@ function generateAPI(controllerMapping, ctx) {
     for (const key of Object.keys(controllerMapping)) {
         const context = controllerMapping[key];
         if (context && context.path && context.verb) {
+            const path = context.path.replace(/\/$/, '');
```

Comment:
Not sure what was the reason for this but I would think it's no longer needed when using `path-to-regexp`. Tests still pass.

Reply:
I think it was to remove a possible '/' on the end of the path